sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -322,3 +322,28 @@ SELECT INTERVAL '153722867280' MINUTE;
 SELECT INTERVAL '-153722867280' MINUTE;
 SELECT INTERVAL '54.775807' SECOND;
 SELECT INTERVAL '-54.775807' SECOND;
+
+SELECT INTERVAL '1' DAY > INTERVAL '1' HOUR;
```

Comment:
let's focus on different fields, not different binary operators.

Reply:
and test `array(...)`, `coalesce(...)` as well.

Reply:
Done

Reply:
> let's focus on different fields, not different binary operators.

I think current can cover all the case?